kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -153,13 +164,16 @@
     data() {
       return {
         userMenuDropdownIsOpen: false,
+        userSyncStatus: [],
```

Comment:
Given that this is expected to be a plain object, using an empty array seems like an error here. Maybe `null` as you are doing a falsey check elsewhere?